packages/react-components/react-card/src/stories/Card/CardSelectable.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { makeStyles, shorthands } from '@fluentui/react-components';
+import { SampleCard } from './SampleCard.stories';
```

Comment:
It's definitely good to promote code reuse... but not here 😅

What happens with stories files is that the source is parsed and passed to our codesandbox export feature. Relative imports like this will just break codesandbox when users try to export the example

Reply:
TODO: create issue to track adding linting for this scenario